src/main/java/Duke/Task.java

```diff
@@ -0,0 +1,69 @@
+package Duke;
+
+/**
+ * The class is an abstract class with parameters task and done
+ * @author LimWeiJun
+ */
+abstract public class Task {
```

Comment:
Hi Wei Jun,
I like how the fields and methods are named. It is easy to tell that `isDone` is a boolean just from its name. The method names also all start with verbs.
However, the name `task` could be more descriptive. May I suggest `description`?